CircleciScripts/replace_android10.py

```diff
@@ -0,0 +1,24 @@
+from utils import replacefiles
+import demjson
+import sys
+
+root = sys.argv[1]
+
+replaces = [
+    {
+        "match" : 'android-23', 
```

Comment:
there needs to be a way to remove the relevant code that uses api 23 and 18. This code is safe-guarded during runtime, but this ensure the rest of the code adheres to api level 10. Check with @kvasukib  which code parts should be removed

Reply:
addressed